### DISCUSSION TITLE
Vegas Pro Edit Compatibility

### DISCUSSION THREAD

**Discussion by 4stacks67** [otomad/OtomadHelper#57] [Q&A] [answered]
Is this compatible with vegas pro edit?

**Comment by Evauation** [comment, marked as answer]
In short: yes!

Vegas Pro Edit is merely a package that contains only Vegas Pro. While Vegas Pro Suite or Vegas Pro Post contain additional software that allows for more flexible editing and features. Vegas Pro Edit is just a software bundle for Vegas Pro. So like all Vegas Pro versions, it does support scripts.

**Reply by 4stacks67** [reply]
Including Post?

**Reply by Evauation** [reply]
Edit, Suite and Post are just packages of the software containing Vegas Pro (the default software) then other various programs that MAGIX has made or licensed to use with Vegas Pro. (e.g. Sound Forge, Vegas Image, Vegas Effects, Mocha)
